### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by JunkFood02** [JunkFood02/Seal#190] [Announcements]
## What's Changed
This version comes with many feature updates and UI/UX improvements.

Feature Updates:

- Add aria2c support
- Export/import command templates to/from clipboard
- SponsorBlock integration
- Cookies settings for downloads
- Better compatibility for custom commands
- Clear temp files

Behavior Changes:

- The default filename is reset to %(title)s [%(id)s].%(ext)s

UI/UX Improvements:

- Multi-selecting operation in Downloads page
- Add monochrome icon support for Android 13

**Full Changelog**: https://github.com/JunkFood02/Seal/compare/v1.1.0...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JunkFood02/Seal/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by Marco-9456** [comment]
Is this meant to be on the stabel release ?
![Screenshot_20220904-181938_Seal](https://user-images.githubusercontent.com/41828058/188323287-68708fd8-a31d-477d-9e96-b4b9aaf878e8.jpg)

**Reply by JunkFood02** [reply]
Forgot to remove it but that's fine 😌👌

**Reply by Marco-9456** [reply]
> Forgot to remove it but that's fine 😌👌

I thought so 🤣

**Comment by BobbyESP** [comment]
What about #194 ? It's not the best way but works... Just for workaround

**Reply by JunkFood02** [reply]
After some testing, I think this issue only appears on devices with specific versions of MIUI. The device I'm using has this problem fixed after an OTA, so I don't see the need to have a patch for such an uncommon bug

**Reply by BobbyESP** [reply]
okey! Thanks for the info

**Reply by BobbyESP** [reply]
> After some testing, I think this issue only appears on devices with specific versions of MIUI. The device I'm using has this problem fixed after an OTA, so I don't see the need to have a patch for such an uncommon bug

Can you share to me what device, MIUI and Android version have you tested?

**Reply by JunkFood02** [reply]
MIUI 13.0.5+ (API 31)
The China version of MIUI should be newer than the international releases I think

**Reply by BobbyESP** [reply]
Yeah, chinese verion of MIUI is better than the EEA/International one
